libjcat/jcat-blob.h

```diff
@@ -19,6 +19,9 @@ G_DECLARE_DERIVABLE_TYPE(JcatBlob, jcat_blob, JCAT, BLOB, GObject)
  * @JCAT_BLOB_KIND_GPG:			GPG detached signature
  * @JCAT_BLOB_KIND_PKCS7:		PKCS-7 detached signature
  * @JCAT_BLOB_KIND_SHA1:		SHA-1 checksum
+ * @JCAT_BLOB_KIND_MANIFEST:		Merkle tree manifest
+ * @JCAT_BLOB_KIND_CHECKPOINT:		Merkle tree checkpoint
+ * @JCAT_BLOB_KIND_INCLUSION_PROOF:	Merkle tree inclusion proof
```

Comment:
@AlCutter I wondered if these need some kind of prefix? e.g. `JCAT_BLOB_KIND_FT_MANIFEST` -- manifest seems like some other verification or signing solution might use, and each kind is supposed to be specific -- e.g. `PKCS7` rather than `PUBKEY_CERTFICATE`.

Reply:
Also, I guess then the description becomes "_Firmware transparency manifest_" too.

Reply:
Ah, I see - yes using the `FT_` prefix seems like a good idea, I like the more specific description too.

Reply:
@AlCutter would `_BT` and `Binary transparency` be a better prefix to use? is there anything firmware specific about what we're doing with Trillian?

Reply:
Yes, that sounds fine - It does seem that Jcat isn't really f/w specific. (I must admit I hadn't come across the MS `catalog` files this is inspired by before!)

Reply:
Okay, done.